src/index.js

```diff
@@ -1,3 +1,28 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
 import 'bootstrap';
-import './styles.css';
\ No newline at end of file
+import './styles.css';
+import weather from './weather';
+import weatherContainer from './weather_container';
+
+const myFunction = () => {
+  const searchButton = document.getElementById('search_button');
+
+  searchButton.addEventListener('click', () => {
+    const searchStart = document.getElementById('search_input').value;
+
+    weather(searchStart)
+      .then((value) => {
+        const mainContainer = document.getElementById('main-container');
+        mainContainer.innerHTML = '';
+
+        mainContainer.appendChild(weatherContainer(value));
+
+        return true;
+      })
+      .catch(() => {
+        alert('Invalid city, please try again!');
```

Comment:
* [ ] Do not use Windows.alert(). It’s ugly, impossible to style, stop code execution, and display differently on different browsers. Use custom modal instead
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/alert_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/javascript-module-v-10/166782/path/step/97634928/)).